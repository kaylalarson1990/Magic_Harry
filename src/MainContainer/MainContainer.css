h1,
p {
  text-align: center;
}

h1 {
  font-size: 68px;
  margin-top: 150px;
}

p {
  font-family: "Open Sans";
  font-size: 28px;
  font-weight: 300;
  color: #be0125;
}

.wand {
  width: 25px;
}
