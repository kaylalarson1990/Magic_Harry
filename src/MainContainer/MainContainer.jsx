import React from "react";
import "./MainContainer.css";
import wand2 from "../images/wand2.png";

export const MainContainer = () => {
  return (
    <section>
      <h1>Welcome to the wizarding world of Harry Potter!</h1>
      <p>Please choose a category above. <img src={wand2} alt="wand" className="wand" /></p>
    </section>
  );
};
