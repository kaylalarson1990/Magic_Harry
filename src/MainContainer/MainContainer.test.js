import React from 'react';
import { shallow } from "enzyme";
import { MainContainer } from './MainContainer';

describe("MainContainer", () => {
    let wrapper;
  
    beforeEach(() => {
      wrapper = shallow(<MainContainer />);
    });
  
    it("should match snapshot", () => {
      expect(wrapper).toMatchSnapshot();
    });
});