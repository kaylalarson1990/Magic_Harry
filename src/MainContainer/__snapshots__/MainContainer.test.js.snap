// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainContainer should match snapshot 1`] = `
<section>
  <h1>
    Welcome to the wizarding world of Harry Potter!
  </h1>
  <p>
    Please choose a category above to begin.
    <img
      alt="wand"
      className="wand"
      src="wand2.png"
    />
  </p>
</section>
`;
